src/actions/pause/mapFrames.js

```diff
@@ -130,6 +132,10 @@ async function expandFrames(
     };
 
     originalFrames.forEach((originalFrame, j) => {
+      if (!originalFrame.location || !originalFrame.thread) {
```

Comment:
Not sure if this is the right approach. From what I read in `XScope.search()`, `displayName` will always be present and `location` may be present but `thread` shouldn't be. However, it was accessed below so perhaps I've misread.